### PR TITLE
Readme update

### PR DESCRIPTION
**Pull request by mortonjt**
This PR is making two minor fixes

* Fixes bug in the main `ag` script related to creating new databases
* Updates the README to clarify the installation process

Both of these issues are discusses in https://github.com/biocore/american-gut-web/issues/388

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,17 +3,43 @@ american-gut-web
...
+
+Time to configure some of the configurations.  Namely, make sure to fill in entries for `USER`, `PASSWORD`, `HOST`.
+
+
+To enable uuid v4 function in postgres:
+```
+echo 'CREATE EXTENSION "uuid-ossp";' | psql
+```
+
+Finally, make sure that all of your permissions are set correctly.  See [ALTER USER](http://www.postgresql.org/docs/9.4/static/sql-alterrole.html)
```
Comment (wasade) [inline README.md]: Is this necessary with a default install?

File: README.md [hunk 2]
```diff
@@ -3,17 +3,43 @@ american-gut-web
...
+
+Finally run the tests to populate the databases and launch the website
```
Comment (wasade) [inline README.md]: Second use of "finally" in close proximity

File: scripts/ag [hunk 3]
```diff
@@ -277,7 +277,8 @@ def patch():
 def _create_database(force=False):
     # Connect to the postgres server
     try:
-        conn = connect(user=AMGUT_CONFIG.user, password=AMGUT_CONFIG.password,
+        conn = connect(dbname='postgres',
```
Comment (wasade) [inline scripts/ag]: can this source from config?

**Comment by wasade** [comment]
@mortonjt, looks like some conflicts?

**Comment by mortonjt** [comment]
A ha.

Ok.  I think I understand psycopg2 a little better.
If you don't have a username, pscyopg2 will not look for a database and will by default connect to `postgres`

If you do have a username (say ag), psycopg2 will try to connect to ag.

So `./scripts/ag make test` breaks if there is a username specified.

File: README.md (hunk 1 above)
Comment (mortonjt) [inline README.md]: Only `DATABASE` is necessary.  Will change in upcoming commit

**Pull request closed without merging**